### PR TITLE
fix: Fixing main entry point in webpack

### PR DESCRIPTION
**Pull request by KillianCourvoisier**

### CODE REVIEW

**Comment by Crash--** [comment]
Can you add a small on_delete.js file to this repo? 🙏 

Thanks <3

**Comment by KillianCourvoisier** [comment]
@Crash--  I can't find a right way to do this, the modifications I made didn't seems to work as intended, as it just kind of remove the Travis building phase.  Any tips would be welcome ...

Second remark, did this need to be marked as green dot or do we prevent the build on the template ?

File: webpack.config.js [hunk 1]
```diff
@@ -28,7 +26,9 @@ try {
 const appIconRX = iconName && new RegExp(`[^/]*/${iconName}`)
 
 module.exports = {
-  entry,
+  entry: {
```
Comment (doubleface) [inline webpack.config.js:29]: Cette modification était nécessaire ?

Comment (KillianCourvoisier) [reply webpack.config.js:29]: C'est ce que m'avait filé Seb, ça m'a été demandé d'y faire par Quentin, je saurais pas te répondre si c'était "nécessaire" mais voila ... :man_shrugging:

**Comment by LucsT** [comment]
@Crash-- What's was the goal of this PR ?

Removing the warning concerning entry point was done by removing entry point in package.json.

If it's for making the template correctly build, in my opinion next step is to create a template connector in registry and allow deploy of dev version. (Or avoid deploy in template case)

So I think we can close that.

**Comment by Crash--** [comment]
> Removing the warning concerning entry point was done by removing entry point in package.json.

I still think this was not the right fix for this issue. It works because npm defaults to `index.js` at the root of the folder. Nothing else. But explicit is much better than implict. 

package.json.main: Is used by npm and yarn in order to know where is the main entry point of the published package. (cf https://docs.npmjs.com/cli/v9/configuring-npm/package-json#main). 

webpack.config: Is here to tell webpack where is the entry point in order to bundle the files. This is absolutely not related to npm package.json and it makes a lot of sense to not depend on it. 
In the meantime we had a shared webpack config, maybe it's there that we have to update the config to use it here 


> If it's for making the template correctly build, in my opinion next step is to create a template connector in registry and allow deploy of dev version. (Or avoid deploy in template case)

By doing the first action, we find out that the CI was red on this repo. So we fixed it by not deploying the template to the registry. 
I still don't know if it's a good idea to want a full deploy to the registry.

**Comment by LucsT** [comment]
Ok understood for the first point to have a explicit entry point declaration, I will work on it. Here and in konnector-build.

Second point, I noted your position about the template in registry.
In this case, we need to have a look probably on cozy-app-publish to avoid deploying to registry in a certain context ( Env var in travis ?, what do you think ?)
The build is ok now only for PR, but not because of avoiding registry deploy.
Maybe we need another channel for this subject.

**Comment by LucsT** [comment]
@Crash-- @KillianCourvoisier 
This commit should define explicitly the entry point.
We probably need to test it anywhere in prod (payfit ?)
And i start a PR on cozy-konnector-build in our mutualised webpack config here :
https://github.com/konnectors/libs/pull/891

What do you think ?

**Pull request merged**